app/plot_app/plotting.py

```diff
@@ -847,9 +853,9 @@ def add_graph(self, field_names, legends, window='hann', window_length=256, nove
 
             # offset = int(((1024/2.0)/250.0)*1e6)
             # scale time to microseconds and add start time as offset
-            time = time * 1.0e6 + self._cur_dataset.data['timestamp'][0]
+            time = time * 1.0e6 + self._cur_dataset.data[timestamp_key][0]
 
-            color_mapper = LinearColorMapper(palette=viridis(256), low=-80, high=0)
+            color_mapper = LinearColorMapper(palette=viridis(256))
```

Comment:
The plot you show now misses the labels on the bar. Is there a way to get that back?
I want to make sure we can still compare. Otherwise you could set an explicit limit per plot.